wpimath/src/main/native/include/frc/kinematics/Odometry.h

```diff
@@ -58,6 +60,38 @@ class WPILIB_DLLEXPORT Odometry {
     m_previousWheelPositions = wheelPositions;
   }
 
+  /**
+   * Resets the pose.
+   *
+   * @param pose The pose to reset to.
+   */
+  void ResetPose(const Pose2d& pose) {
+    m_gyroOffset =
+        Rotation2d{m_gyroOffset + (pose.Rotation() - m_pose.Rotation())};
```

Comment:
Huh, didn't realize C++ won't automatically define `operator+=` in terms of `operator+`. Anyways, is it necessary to wrap this in `Rotation2d{}`? (e.g., would `m_gyroOffset = m_gyroOffset + (pose.Rotation() - m_pose.Rotation())` work?)

Reply:
Potentially iirc Rotation2d doesn't do operator overloading I could definitely be wrong though

Reply:
Ohh yeah whoops that should be fine